bundle/jsky.app.ot/src/main/scala/jsky/app/ot/editor/seq/ItcTable.scala

```diff
@@ -222,33 +222,39 @@ trait ItcTable extends Table {
       wl  <- ConfigExtractor.extractObservingWavelength(c)
       mag <- mags(wl)
     } yield {
-      val b = mag.getBrightness
-      mag.getBand match {
-        // TODO: unify band definitions from spModel core and itc shared so that we don't need this translation anymore
-        case Magnitude.Band.u  => (b, WavebandDefinition.U)
-        case Magnitude.Band.g  => (b, WavebandDefinition.g)
-        case Magnitude.Band.r  => (b, WavebandDefinition.r)
-        case Magnitude.Band.i  => (b, WavebandDefinition.i)
-        case Magnitude.Band.z  => (b, WavebandDefinition.z)
-
-        case Magnitude.Band.U  => (b, WavebandDefinition.U)
-        case Magnitude.Band.B  => (b, WavebandDefinition.B)
-        case Magnitude.Band.V  => (b, WavebandDefinition.V)
-        case Magnitude.Band.R  => (b, WavebandDefinition.R)
-        case Magnitude.Band.I  => (b, WavebandDefinition.I)
-        case Magnitude.Band.Y  => (b, WavebandDefinition.z)
-        case Magnitude.Band.J  => (b, WavebandDefinition.J)
-        case Magnitude.Band.H  => (b, WavebandDefinition.H)
-        case Magnitude.Band.K  => (b, WavebandDefinition.K)
-        case Magnitude.Band.L  => (b, WavebandDefinition.L)
-        case Magnitude.Band.M  => (b, WavebandDefinition.M)
-        case Magnitude.Band.N  => (b, WavebandDefinition.N)
-        case Magnitude.Band.Q  => (b, WavebandDefinition.Q)
+      val value  = mag.getBrightness
+      // TODO: unify band definitions from spModel core and itc shared so that we don't need this translation anymore
+      val system = mag.getSystem match {
+        case Magnitude.System.Vega  => BrightnessUnit.MAG
+        case Magnitude.System.AB    => BrightnessUnit.ABMAG
+        case Magnitude.System.Jy    => BrightnessUnit.JY
+      }
+      val band   = mag.getBand match {
+        case Magnitude.Band.u  => WavebandDefinition.U
```

Comment:
I think we discussed this before but this looks so much like the old band to new band conversion in `ModelConverters`, do we need separate models in ITC?